### PR TITLE
feat: add default feed for users

### PR DESCRIPTION
**Pull request by rebelchris**
Add default feed for users and expose id in boot.
Made it relationship to have cascasing, not sure we need anything besides ID?

### CODE REVIEW

File: src/entity/user/User.ts [hunk 1]
```diff
@@ -17,6 +18,7 @@ import type { UserStreakAction } from './UserStreakAction';
 import type { UserCompany } from '../UserCompany';
 import type { UserTopReader } from './UserTopReader';
 import type { SubscriptionCycles } from '../../paddle';
+import { Feed } from '../Feed';
```
Comment (capJavert) [inline src/entity/user/User.ts:21]:
```suggestion
import type { Feed } from '../Feed';
```

File: src/migration/1733987625520-UserDefaultFeed.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class UserDefaultFeed1733987625520 implements MigrationInterface {
+  name = 'UserDefaultFeed1733987625520';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`ALTER TABLE "user" ADD "defaultFeedId" text`);
+    await queryRunner.query(
+      `ALTER TABLE "user" ADD CONSTRAINT "UQ_ff0f49b797aca629f81cef47610" UNIQUE ("defaultFeedId")`,
```
Comment (capJavert) [inline src/migration/1733987625520-UserDefaultFeed.ts:9]: why unique constraint?

File: src/routes/boot.ts [hunk 3]
```diff
@@ -420,6 +420,7 @@ const getUser = (
       'followingEmail',
       'followNotifications',
       'subscriptionFlags',
+      'defaultFeedId',
```
Comment (capJavert) [inline src/routes/boot.ts:423]: also add to `UpdateUserInput` so we can set it on frontend?

File: src/migration/1733987625520-UserDefaultFeed.ts (hunk 2 above)
Comment (rebelchris) [reply src/migration/1733987625520-UserDefaultFeed.ts:9]: Ah missed it, just something ORM generated I guess 😅

File: src/entity/user/User.ts (hunk 1 above)
Comment (rebelchris) [reply src/entity/user/User.ts:21]: Really need to fix webstorm auto import type :jenkins:

File: src/routes/boot.ts (hunk 3 above)
Comment (rebelchris) [reply src/routes/boot.ts:423]: Added, but no specific test yet

**Review by capJavert — APPROVED** [review]

**Review by omBratteng — APPROVED** [review]

**Pull request merged**